specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2106 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Microsoft Cognitive Language Service - Text Analytics",
+        "description": "The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>.0",
+        "version": "2021-11-01-preview"
+    },
+    "securityDefinitions": {
+        "apim_key": {
+            "type": "apiKey",
+            "name": "Ocp-Apim-Sumscription-Key",
+            "in": "header"
+        }
+    },
+    "security": [
+        {
+            "apim_key": []
+        }
+    ],
+    "x-ms-parameterized-host": {
+        "hostTemplate": "{Endpoint}/language",
+        "useSchemePrefix": false,
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/Endpoint"
+          }
+        ]
+    },
+    "paths": {
+        "/analyze-text/jobs": {
```

Comment:
@rokulka What is the latest guideline regarding the colon thing? Is this form OK?